### ISSUE TITLE
Apache 2.4.27 crashes often when restarting the service. segfault in libdl-2.24.so or apache2

### ISSUE THREAD

**Issue by pantaluna** [oerdnj/deb.sury.org#707]
Hi,
Environment:
- O.S.: Ubuntu 17.04 Zesty (Desktop edition)
- Ubuntu PPA: ppa:ondrej/apache2
- Ubuntu Package: *the latest* apache2-bin:amd64/zesty 2.4.27-5.1+ubuntu17.04.1+deb.sury.org+1 

Details:
I have been struggling for quite some time now on a dedicated dev server with Apache "sometimes" crashing a) When the Apache service is started during a server boot; and b) When the Apache service is restarted. I think the 2 problems are related.

It will crash at least once out of +- 25 times when I restart the Apache service (see below for the test script).

Syslog excerpt: the segfault varies from time to time. These are the 2 variations:
````
Sep 22 13:41:57 s3black kernel: [43532.685282] apache2[12965]: segfault at 7f3079b9dad0 ip 00007f3079b9dad0 sp 00007ffe3b47be88 error 14 in libdl-2.24.so[7f307c627000+3000]
...
Sep 22 13:49:28 s3black kernel: [43983.678591] apache2[16930]: segfault at 7f0f0387d8c0 ip 000056527483fdd4 sp 00007ffd4282ba10 error 4 in apache2[565274803000+9e000]
````

I can reproduce it consistently using this script:
````
#@doc The server will crash at least once every +-10 minutes when this script is running.
while true
do
    systemctl stop apache2; systemctl start apache2; journalctl --lines=50 --unit=apache2; sleep 7;
done
````

I prefer not to duplicate all information over here so please go to the Apache problem report for more details such as a pkg versions, syslog, coredump, backtrace, ... at https://bz.apache.org/bugzilla/show_bug.cgi?id=61558

How to proceed?

**Comment by pantaluna** [comment]
A new patch is available at https://bz.apache.org/bugzilla/show_bug.cgi?id=61558
Is it possible to put this in a fresh build of the Apache2 PPA (production or qa-test) so I can retest it?
Thanks.

**Comment by oerdnj** [comment]
Fixed in `2.4.27-6.1`
